frontend/packages/core/src/Feedback/tooltip.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { TooltipProps as MuiTooltipProps } from "@material-ui/core";
+import { Tooltip as MuiTooltip } from "@material-ui/core";
+
+const BaseTooltip = ({ className, ...props }: MuiTooltipProps) => (
+  <MuiTooltip classes={{ tooltip: className }} {...props} />
+);
+
+const StyledTooltip = styled(BaseTooltip)((props: { maxWidth?: string }) => ({
+  maxWidth: props.maxWidth,
+  backgroundColor: "#0D1030",
+  borderRadius: "6px",
+  "&.MuiTooltip-tooltipPlacementLeft": {
+    margin: "0 2px",
+  },
+  "&.MuiTooltip-tooltipPlacementRight": {
+    margin: "0 2px",
+  },
+  "&.MuiTooltip-tooltipPlacementTop": {
+    margin: "2px 0",
+  },
+  "&.MuiTooltip-tooltipPlacementBottom": {
+    margin: "2px 0",
+  },
+}));
+
+export interface TooltipProps extends Pick<MuiTooltipProps, "interactive" | "placement"> {
```

Comment:
fyi `interactive` prop is for making the tooltip not disappear when moving the mouse to it